examples/basic/main.tf

```diff
@@ -113,6 +113,8 @@ module "observability_agents" {
   logdna_agent_tags         = var.resource_tags
   logdna_add_cluster_name   = true
   # example of how to include / exclude metrics - more info https://cloud.ibm.com/docs/monitoring?topic=monitoring-change_kube_agent#change_kube_agent_log_metrics
-  sysdig_metrics_filter = [{ type = "exclude", name = "metricA.*" }, { type = "include", name = "metricB.*" }]
-  sysdig_agent_tags     = var.resource_tags
+  sysdig_metrics_filter              = [{ type = "exclude", name = "metricA.*" }, { type = "include", name = "metricB.*" }]
+  sysdig_agent_tags                  = var.resource_tags
+  logdna_agent_custom_line_exclusion = var.logdna_agent_custom_line_exclusion
+  logdna_agent_custom_line_inclusion = var.logdna_agent_custom_line_inclusion
```

Comment:
hard code values for these instead of using new variables in the example. you can write code comments to explain them if you wish. You should also update the example readme to explain the example also shows how to add custom line exclusions and inclusions.